week2/Week2.py

```diff
@@ -0,0 +1,140 @@
+# Task 1 : Checking type of number
+
+def fizz_buzz(num):
+    if num % 3 == 0 and num % 5 == 0:
+        print(num, "-", "Fizz Buzz")
+    elif num == 1 or num == 2 or num == 3 or num == 5:
+        print(num, "-", "Prime")
+    elif num % 5 == 0 and num != 5:
+        print(num, "-", "Buzz")
+    elif num % 3 == 0 and num != 3:
+        print(num, "-", "Fizz")
+    elif num % 2 != 0:
+        for i in range(3, 100):
+            if (num % i) != 0:
+                print(num, "-", "Prime")
+            break
+    else:
+        print(num)
+
+
+fizz_buzz(9)
+
+# Task 2 : try and except error for index
+
+
+try:
+    random = [5, 10, 20]
+    print(random[3])
+except IndexError as e:
+    print(e)
+
+
+# Task 3 : Create a class of Jet inventory with two arguments i.e name and country. Also add the minimum 2 items in
+# the class and print them.
+
+class Jet:
+    def __init__(self, name, country):
+        self.jet_name = name
+        self.country = country
+
+    def details(self):
+        print(self.jet_name, self.country)
+
+
+new_jet = Jet("Jet Airways", "India")
+new_jet.details()
+
+# Task 5 - Write a Python script to check whether a given key already exists in a dictionary.
+
+SongInfo = {"1": "album", "2": "rating", "3": "singer", "4": "writer", "5": "musician",
```

Comment:
please stick to `snake_case`with your variable names :) don't mix. `SongInfo`is neither `snake_case`nor `camelCase`

Reply:
Thanks. It is big mistake from my side always. Next time I will take care of this variables naming.